BUILDING.md

```diff
@@ -14,7 +16,7 @@ All of them are listed in the following installation command:
 
 ```
 Linux:
-sudo apt-get update && sudo apt-get install build-essential git gperf libgmp-dev cmake bison curl flex gcc-multilib linux-libc-dev libboost-all-dev libtinfo-dev ninja-build python3-setuptools unzip wget
+sudo apt-get update && sudo apt-get install build-essential git gperf libgmp-dev cmake bison curl flex gcc-multilib linux-libc-dev libboost-all-dev libtinfo-dev ninja-build python3-setuptools unzip wget python3-pip openjdk-8-jre
 
```

Comment:
Why did you need `openjdk-8-jre` package? Are you sure about it?

Reply:
This is because of an error that happened while setting up CVC4 solver, and I preferred to add it on the "Installing Dependencies" step rather than on "Setting Up CVC4" step.
![image](https://user-images.githubusercontent.com/37079552/105355167-5cb13100-5bfa-11eb-808f-686bfc19423b.png)

Reply:
Yeah I agree, nice finding!